redis/redis.go

```diff
@@ -118,3 +116,32 @@ func (s *Storage) Close() error {
 func (s *Storage) Conn() redis.UniversalClient {
 	return s.db
 }
+
+// Return all the keys
+func (s *Storage) Keys() ([][]byte, error) {
+	var keys [][]byte
+	var cursor uint64
+	var err error
+
+	for {
```

Comment:
wdyt about using iterator instead like this?
```
iter := rdb.Scan(ctx, 0, "prefix:*", 0).Iterator()
for iter.Next(ctx) {
	fmt.Println("keys", iter.Val())
}
if err := iter.Err(); err != nil {
	panic(err)
}
```

Reply:
I could but my implementation is based on Redis official documentation for using `scan()`: 

See here: https://pkg.go.dev/github.com/redis/go-redis/v9#Client.Scan